percy-common.gemspec

```diff
@@ -21,9 +21,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'excon', '~> 0.57'
   spec.add_dependency 'redis', '>= 4.1.3', '< 5.0.0'
 
-  spec.add_development_dependency 'bundler', '~> 2.1.4'
+  spec.add_development_dependency 'bundler', '~> 2.0'
   spec.add_development_dependency 'guard-rspec', '~> 4.7'
   spec.add_development_dependency 'percy-style', '~> 0.7.0'
   spec.add_development_dependency 'rake', '~> 13.0'
   spec.add_development_dependency 'rspec', '~> 3.2'
+  spec.add_development_dependency 'webrick', '~> 1.0'
```

Comment:
`webrick` doesn't appear to be included in Ruby 3. See https://github.com/jekyll/jekyll/issues/8523#issuecomment-751409319